policies/pull-request-review-and-merge.md

```diff
@@ -0,0 +1,60 @@
+# Policy on pull request review and merging in the Astropy Project
+
+## Astropy core
+
+In the astropy core package the requirements are as follows:
+
+- At least one contributor with relevant expertise must review the pull request
+  in detail.
+  - New or updated code is logically correct and achieves the desired effect of
```

Comment:
Should we change the indentation from 2-space to 4-space? At first, I thought the indentation is a typo.

Reply:
I think it doesn't matter, but in markdown I always use 2.